projectq/backends/_ibm/_ibm_test.py

```diff
@@ -110,9 +144,10 @@ def test_ibm_backend_functional_test(monkeypatch):
                     '\\nmeasure q[0] -> c[0];\\nmeasure q[2] -> c[2];\\nmeas'
                     'ure q[1] -> c[1];"}], "shots": 1024, "maxCredits": 5,'
                     ' "backend": {"name": "simulator"}}')
-
+    correct_info = ('{"qasms": [{"qasm": "\\ninclude \\"qelib1.inc\\";\\nqreg q[3];\\ncreg c[3];\\nh q[0];\\ncx q[0], q[2];\\nh q[1];\\ncx q[1], q[2];\\nh q[2];\\ntdg q[2];\\nsdg q[2];\\nh q[0];\\nh q[1];\\nbarrier q[2], q[0], q[1];\\nu3(0.2, -pi/2, pi/2) q[2];\\nmeasure q[2] -> c[2];\\nmeasure q[0] -> c[0];\\nmeasure q[1] -> c[1];"}], "shots": 1024, "maxCredits": 5, "backend": {"name": "simulator"}}')
```

Comment:
`projectq/backends/_ibm/_ibm_test.py:147:80: E501 line too long (405 > 79 characters)`

There are a few other pep8 things in this PR

Reply:
Thanks! Fixed.